config/webpack/environment.js

```diff
@@ -1,12 +1,16 @@
 const { environment } = require('@rails/webpacker')
+const webpack = require('webpack')
 
-let ASSET_HOST
-
-if (process.env.NODE_ENV === 'production') {
-  ASSET_HOST =
-    'https://exercism-assets-staging.s3.eu-west-2.amazonaws.com/packs/'
-} else {
-  ASSET_HOST = ''
-}
+environment.plugins.prepend(
+  'Environment',
+  new webpack.EnvironmentPlugin(
+    JSON.parse(
+      JSON.stringify({
+        // TODO: @iHiD Should we get this from Exercism::Config?
+        BUGSNAG_API_KEY: '938ae3d231c5455e5c6597de1b1467af',
```

Comment:
Yes please. I'll merge this for now, but then put a PR together that reads it from Exercism::Config and I'll do the magic in the AWS end to get it all working.